SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.cc

```diff
@@ -43,8 +43,14 @@ namespace {
   constexpr double m_electron = 0.511;
   constexpr double m_muon = 105.658;
   constexpr double m_proton = 938.272;
+  double calcQ(float x);
+}  // namespace
+namespace {
+  double calcQ(float x) {
+    auto xx = std::min(0.5f * x * x, 12.5f);
+    return 0.5 * (1.0 - std::copysign(std::sqrt(1.f - unsafe_expf<4>(-xx * (1.f + 0.2733f / (1.f + 0.147f * xx)))), x));
```

Comment:
use named constants rather than magic numbers, or at least provide a comment explaining what the numbers mean